src/config/mod.rs

```diff
@@ -393,25 +401,48 @@ mod tests {
             .build()
             .unwrap();
 
-        assert_eq!(config.initrd_path.unwrap().as_os_str(), "/tmp/initrd.img");
+        assert_eq!(
+            config.initrd_path.as_ref().unwrap().as_os_str(),
+            "/tmp/initrd.img"
+        );
```

Comment:
why is this change needed? :thinking:

Reply:
Looking at the next commit, I realized you probably made a mistake with splitting. This change isn't needed in this commit (probably some others too).